utils/interactive.js

```diff
@@ -409,16 +410,38 @@ $.extend(KhanUtil.Graphie.prototype, {
         var temp = [];
 
         if (Math.abs(angle - 90) < 1e-9 && options.showRightAngleMarker) {
-            // Draw right angle box
-            var v1 = addPoints(sVertex, scaledPolarDeg(sRadius, startAngle));
-            var v2 = addPoints(sVertex, scaledPolarDeg(sRadius, endAngle));
-
-            sRadius *= Math.SQRT2;
-            var v3 = addPoints(sVertex, scaledPolarDeg(sRadius, halfAngle));
-
-            _.each([v1, v2], function(v) {
-                temp.push(graphie.scaledPath([v, v3], options.style));
-            });
+            if (options.dottedRightAngleMarker) {
+                // Draw EU style right angle notation, arc with a dot
+                temp.push(graphie.arc(
+                    vertex,
+                    graphie.unscaleVector(sRadius),
+                    startAngle,
+                    endAngle,
+                    options.style
+                ));
+                // Calculate angle bisector
+                var a = kvector.normalize(kvector.subtract(p1, vertex));
+                var b = kvector.normalize(kvector.subtract(p3, vertex));
+                var bisector = kvector.normalize(addPoints(a, b));
+                // Place the dot in the middle of vertex and arc
+                bisector = kvector.scale(bisector, graphie.unscaleVector(sRadius)[0] * 0.5);
+                var dotCenter = addPoints(vertex, bisector);
+                var dotSize = 0.5;
+                temp.push(graphie.ellipse(
+                    dotCenter,
+                    [dotSize / graphie.scale[0], dotSize / graphie.scale[1]],
+                    _.extend({"fill-opacity": 1, "stroke-width": 1}, options.style)
+                ));
```

Comment:
Can we add `sRadius *= Math.SQRT2;` to preserve the same spacing between `label` for both drawing options?